domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ccj/PaymentSchedule.java

```diff
@@ -1,9 +1,9 @@
 package uk.gov.hmcts.cmc.domain.models.ccj;
 
 public enum PaymentSchedule {
-    EACH_WEEK("each week"),
-    EVERY_TWO_WEEKS("every two weeks"),
-    EVERY_MONTH("every month");
+    EACH_WEEK("Each week"),
```

Comment:
was anything else using this text in lower case?

Reply:
Haven't spotted anything which is using description. CCD equivalent class is made same as this.